src/views/CavaticaProjectsView.js

```diff
@@ -62,6 +63,9 @@ const CavaticaProjectsView = () => {
   if (error)
     return (
       <Container as={Segment} basic>
+        <Helmet>
+          <title>KF Data Tracker - Cavatica projects - Error</title>
```

Comment:
same as above _maybe_ add study id

Reply:
This is the full Cavatica projects list for ADMIN role, I will add study ID to all others under a study.